tests/testthat/test-row-to-names.R

```diff
@@ -223,3 +248,34 @@ test_that("find_header works within row_to_names", {
     setNames(find_correct[4:nrow(find_correct),], c("D", "E"))
   )
 })
+
+test_that("multiple rows input works", {
+  q_row_to_names <- purrr::quietly(row_to_names)
```

Comment:
Rather than using `q_row_to_names`, can you please wrap the individual tests in `suppressMessages()`?  For my way of reading code, that makes tests simpler to understand.

Reply:
Done.
I went with `suppressWarnings()` since this was a warning rather than a message.